src/main/webapp/dev/js/main/adminSearch.es6

```diff
@@ -144,4 +145,56 @@ $(document).ready(() => {
+        const studentName = $entry.find('input[name=studentName]').val();
+        const sessionStatus = $entry.find('input[name=sessionStatus]').val();
+        const sessionName = $entry.find('#feedback-session-name').text();
+        const sessionNameWithoutDate = sessionName.substring(0, sessionName.lastIndexOf(' ['));
+
+        const relatedLink = encodeURIComponent($entry.find('input[name=relatedLink]').val());
+        const googleSignup = encodeURIComponent('https://accounts.google.com/NewAccount');
+        const supportEmail = `${Const.TEAMMATES_EMAIL_ADDRESS}`;
+
+        const bodyHeader = `Hello ${studentName},`;
+        const bodyFooter =
+                '%0D%0A%0D%0AIf you encounter any problems when using the system,'
+                + ` you can email TEAMMATES support team at ${supportEmail}.`
+                + '%0D%0A%0D%0ARegards,%0D%0ATEAMMATES Team.';
+
+        const uniqueLinkMessage = 'The above link is unique to you. Please do not share it with your classmates.';
+
+        let subject = '';
+        let bodyContent = '';
+
+        if (subjectType === 'Invitation to join course') {
+            subject = `TEAMMATES: ${subjectType} [${courseName}][Course ID: ${courseId}]`;
+            bodyContent =
+                    `%0D%0A%0D%0AThe course ${courseName} is using the TEAMMATES System to collect feedback.`
+                    + ` %0D%0A%0D%0ATo "join" the course, please go to this Web address: ${relatedLink}`
+                    + ' %0D%0A%0D%0A*If prompted to log in, use your Google account to log in. If you do not have'
+                    + ` a Google account, please create one from the Google Accounts page: ${googleSignup}`
+                    + ` %0D%0A*${uniqueLinkMessage} %0D%0A%0D%0ANote that If you wish to access TEAMMATES`
+                    + ' without using your Google account, you do not need to ‘join’ the course as instructed above.'
+                    + ' You will still be able to submit/view feedback by following the instructions sent to you by'
+                    + ' TEAMMATES at the appropriate times. However, we recommend joining the course using your Google'
+                    + ' account, because it gives you more convenient access to all your feedback stored in TEAMMATES.';
+        } else {
+            subject = `TEAMMATES: ${subjectType} [Course: ${courseName}][Feedback Session: ${sessionNameWithoutDate}]`;
+            bodyContent =
+                    `%0D%0A%0D%0AThe following feedback session is ${sessionStatus}.%0D%0A`
+                    + `Course: [${courseId}][${courseName}]%0D%0AFeedback Session Name: ${sessionName}%0D%0A`
+                    + '%0D%0AThe link to the feedback for the above session, please go to this Web'
+                    + ` address: ${relatedLink} %0D%0A%0D%0A*${uniqueLinkMessage}`;
+        }
+
+        const emailWrapper = `mailto:${studentEmail}?Subject=${subject}&body=${bodyHeader}${bodyContent}${bodyFooter}`;
```

Comment:
A query: is it possible for the course name or other fields to include unsafe characters e.g. `&body=` that would disrupt the URL encoding?

Reply:
@whipermr5 I tried doing that but didn't find any flaw in that everything works fine in that case as well, but I don't know why! :sweat_smile:

Reply:
What's the corresponding mailto link generated?

Reply:
Sorry for the delay :sweat_smile: 
![screenshotfrom2017-10-1014-08-27](https://user-images.githubusercontent.com/9383897/31377169-c8fe0ea6-adc4-11e7-85bf-15d674475129.jpg)

Reply:
The unsafe fields need to be URL encoded first.

Reply:
@whipermr5 I think there are no unsafe fields